assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -130,6 +130,12 @@ che.docker.always_pull_image=true
 # to be able to launch their own runtimes which are embedded Docker containers.
 che.docker.privilege=false
 
+# Adds options when mounting the /projects volume.
+che.machine.docker.volumes.projects.options=Z
+
+# Adds options when mounting the /mnt/che/terminal, /mnt/che/ws-agent.tar.gz, /mnt/che/conf volume
+che.machine.docker.volumes.workspace.options=ro,Z
```

Comment:
According to a comment it related to agents rather to workspace. Probably property name should be aligned with the comment.